concent-vm/helper-scripts/concent-run.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash -e
+
+trap "kill 0" SIGINT
```

Comment:
Please add a comment explaining what this command does. It goes a bit beyond casual shell scripting so you should keep in mind that your script can be read by someone who does not know what you're doing here.

Reply:
Done.